src/components/KMultiselect/KMultiselect.vue

```diff
@@ -32,96 +29,101 @@
             return
           }"
           :position-fixed="positionFixed"
-          :target="`[id='${multiselectInputId}']`"
-          :test-mode="!!testMode || undefined"
+          target=".multiselect-wrapper"
           @closed="() => handleToggle(false, isToggled, toggle)"
           @opened="() => handleToggle(true, isToggled, toggle)"
         >
           <div
-            ref="multiselectRef"
-            class="k-multiselect-trigger"
-            :class="{ focused: isFocused, hovered: isHovered, disabled: isDisabled, readonly: isReadonly }"
-            data-testid="k-multiselect-trigger"
+            ref="multiselectElement"
+            class="multiselect-trigger"
+            :class="{ focused: isFocused, hovered: isHovered, disabled: isDisabled, readonly: isReadonly, 'multiselect-error': error }"
```

Comment:
question: why do we need `multiselect-error` on this element as well if it's already applied to the outermost parent?

Reply:
To keep styling section simpler (otherwise too much nesting when targeting elements sometimes)

Reply:
```scss
.multiseelct-error .multiselect-trigger {
  // code
}
```
That seems simple..?